i18n/en/docusaurus-plugin-content-docs/current/intro.mdx

```diff
@@ -23,18 +23,41 @@ FSD doesn't enforce a particular programming language, UI framework or state man
 
 If you have an existing project, fear not — FSD can be adopted incrementally. Just make sure that your team is **in&nbsp;pain** from the current architecture, otherwise a switch might not be worth it. <!-- For migration guidance, see the Migration section. TODO: add the migration section and link to it here -->
 
-## Basics
+## Overview
 
-In radically simplified terms, FSD boils down to the following:
-
-- Project partitioning on business domains
-- Specific and consistent file structure (see the image below)
+In FSD, a project consists of _layers_, each layer is made up of _slices_ and each slice is made up of _segments_. 
 
 ![themed--scheme](/img/visual_schema.jpg)
 
-The terms in the diagram above are explained in detail in the [**Abstractions**][refs-basics--abstractions] section.
+The layers are standardized across all projects and vertically arranged. Modules on one layer can only interact with modules from the layers strictly below. There are currently seven of them (bottom to top):
+
+1. **Shared** — reused functionality, detached from the specifics of the project/business.
+2. **Entities** — business entities (e.g., user, product, order).
+3. **Features** — user interactions, actions that bring business value to the user.
+4. **Widgets** — compositional layer to combine entities and features into meaningful blocks.
+5. **Pages** — compositional layer to construct full pages from entities, features and widgets.
+6. **Processes** — complex inter-page scenarios (e.g., authentication). 
+7. **App** — app-wide settings, styles and providers.
+
+Then there are slices, which partition the code by business domain. This makes your codebase easy to navigate by keeping logically related modules close together. Slices cannot use other slices on the same layer, and that helps with high cohesion and low coupling.
+
+Each slice, in turn, consists of segments. These are tiny modules that are meant to help with separating code within a slice by its technical purpose. The most common segments are `ui`, `model`, `api` and `lib`, but you can omit some or add more, as you see fit.
```

Comment:
@azinit we decided to not recommend keep api in slices, am I right? Maybe it's time to finally show it in docs ( I mean, by not mentioning this segment here 😄 )

Reply:
Whaaaaat?) That's the first time I'm hearing this

Reply:
@Rin-Akaia-eth Тож впервые слышу про это)

Reply:
Мы говоорили что api лучше держать в shared, но если это какой-нить graphql - то api рядом как раз норм держать
(целый воркшоп об этом был)

Reply:
GraphQL query description is actually a part of model 😬

But ok, nvm for now, we have to discuss it later again with core team, because it seems like everyone has forgotten that discussion